### PR TITLE
Create index.html

### PR DESCRIPTION
**Pull request by SpookyBugie**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (SpookyBugie) [inline index.html:5]: Why did you add this, doesnt seem to be needed?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>myWebPage</title>
+</head>
+<body>
+    <h1>Hello wrold</h1>
```
Comment (SpookyBugie) [inline index.html:10]: You shouldnt use h1 here, you should use h2

**Review by kamperud — APPROVED** [review]
Lgtm
